packages/layers/src/multiscale-image-layer/multiscale-image-layer.js

```diff
@@ -98,11 +142,22 @@ const MultiscaleImageLayer = class extends CompositeLayer {
          * return type, and optional throw for performance.
          */
         const tiles = await Promise.all(selections.map(getTile));
-
         const tile = {
-          data: tiles.map(d => d.data),
-          width: tiles[0].width,
-          height: tiles[0].height
+          data: tiles.map(d =>
+            clipper.clip({
+              data: d.data,
+              width: tiles[0].width,
+              height: tiles[0].height
+            })
```

Comment:
Is there a reason why we need to use the first tile over and over?

```suggestion
          data: tiles.map(clipper.clip),
```